zjit/src/hir.rs

```diff
@@ -299,6 +299,7 @@ pub enum Const {
     CInt8(i8),
     CInt16(i16),
     CInt32(i32),
+    Shape(ShapeId),
```

Comment:
Curiosity: Why does this one not have the C prefix to match the type like all the others?